File: src/main/java/sawyern/cookiebot/repository/CookieRepository.java
package sawyern.cookiebot.repository;

import org.springframework.data.repository.CrudRepository;
import sawyern.cookiebot.models.entity.Cookie;

import java.util.Collection;

public interface CookieRepository extends CrudRepository<Cookie, String> {
    Collection<Cookie> findByAccountDiscordId(String discordId);
}

File: src/main/kotlin/sawyern/cookiebot/repository/CookieRepository.kt
package sawyern.cookiebot.repository

import org.springframework.data.jpa.repository.JpaRepository
import sawyern.cookiebot.models.Cookie
import sawyern.cookiebot.models.Season

interface CookieRepository: JpaRepository<Cookie, String> {
    fun findByAccountDiscordId(discordId: String): Collection<Cookie>
    fun findByAccountDiscordIdAndSeason(discordId: String, season: Season): Collection<Cookie>
}